driver/csiplugin/controllerserver.go

```diff
@@ -61,6 +62,8 @@ const (
 	pvcNamespaceKey = "csi.storage.k8s.io/pvc/namespace"
 
 	defaultS3Port = "443"
+
+	cacheTempDirName = ".cachevolumetmp"
```

Comment:
This directory has to be passed with --tmpdir option while creating cache volume.

Reply:
addressed